[Process] Introducing a new `PhpSubprocess` handler (#48485)
symfony/symfony: src/Symfony/Component/Process/CHANGELOG.md

CHANGELOG
=========

> 6.3

Reviewer: ```suggestion
6.4
```

-----

>  * added `PhpSubprocess` to handle PHP subprocesses that take over the

Reviewer 2: ```suggestion
6.3
---

 * Add `PhpSubprocess` to handle PHP subprocesses that take over the
```

Author: Is that a policy change? I used the same style as all the other versions in that file on purpose 😊

Reviewer 2: Yes it is, but quite some time ago 👍

Reviewer 3: We changed the style in Symfony 5.3 or 5.4

Reviewer 2: For the record:

Fabien documented how to write `CHANGELOG.md` files here: https://github.com/symfony/symfony-docs/pull/14830

6.4
---

 * Add `PhpSubprocess` to handle PHP subprocesses that take over the
   configuration from their parent
 * Add `RunProcessMessage` and `RunProcessMessageHandler`
 * Support using `Process::findExecutable()` independently of `open_basedir`

> 

Reviewer 3: this extra line should not be added

> 

Reviewer: ```suggestion
```
